Cartfile

```diff
@@ -1,4 +1,4 @@
 github "aws/aws-sdk-ios" ~> 2.13.0
 github "stephencelis/SQLite.swift" ~> 0.12.2
 github "ashleymills/Reachability.swift" ~> 5.0.0
-github "daltoniam/starscream" ~> 3.0.2
```

Comment:
tried installing carthage dependencies in new project with just this line
issues because repo is not found, line needs to be updated to:

github "aws-amplify/aws-appsync-realtime-client-ios"